src/test/java/uk/gov/companieshouse/charges/data/service/ChargesServiceTest.java

```diff
@@ -378,22 +377,17 @@ void testInsertChargeHandlesCompensatoryTransactionWhenServiceUnavailableThrown(
 
         // When
         Executable executable = () -> chargesService.upsertCharges(contextId, companyNumber,
-                chargeId, internalChargeApi);
+                chargeId, buildInternalCharges(OffsetDateTime.parse("2023-11-06T15:30:00.000000Z")));
```

Comment:
Since the compensatory transaction has been removed we no longer need the this test